### PR TITLE
feat: add `repology` variable to all pacscripts maintained by @echometerain

### PR DESCRIPTION
**Pull request by echometerain**

### CODE REVIEW

File: packages/pdf-mix-tool-git/pdf-mix-tool-git.pacscript [hunk 1]
```diff
@@ -7,6 +7,7 @@ breaks="${_name} ${_name}-bin ${_name}-app ${_name}-deb"
 description="a GUI application to split, merge, rotate and mix PDF files"
 optdepends=("ImageMagick: image manipulation tool")
 maintainer="echometerain <echometer@disroot.org>"
+repology="pdfmixtool"
```
Comment (wizard-28) [inline packages/pdf-mix-tool-git/pdf-mix-tool-git.pacscript:10]: Not eligible for `pacup`, so remove this.

File: packages/smartgit/smartgit.pacscript [hunk 2]
```diff
@@ -6,6 +6,7 @@ replace="alacritty"
 hash="c3e2672e98190cb72060e27fa26232b7f4b6ffcf62eaf1d01e7ec94ac1d36683"
 maintainer="echometerain <echometer@disroot.org>"
 removescript="true"
+repology="smartgit"
```
Comment (wizard-28) [inline packages/smartgit/smartgit.pacscript:9]:
```suggestion
repology=("project: smartgit")
```

File: packages/vega-strike-git/vega-strike-git.pacscript [hunk 3]
```diff
@@ -4,6 +4,7 @@ build_depends="make"
 description="Space Flight Simulator and trading game"
 pacdeps="vega-strike-engine-git"
 maintainer="echometerain <echometer@disroot.org>"
+repology="vegastrike-data"
```
Comment (wizard-28) [inline packages/vega-strike-git/vega-strike-git.pacscript:7]: Not eligible for `pacup`

File: packages/yad/yad.pacscript [hunk 4]
```diff
@@ -11,6 +11,7 @@ optdepends=("libwebkit2gtk-4.0-dev: for supporting HTML dialog"
   "libgtksourceview-4-dev: for enabling syntax highlighting in text-info dialog"
   "libgspell-1-dev: for support spell checking in text fields")
 maintainer="echometerain <echometer@disroot.org>"
+repology="yad"
```
Comment (wizard-28) [inline packages/yad/yad.pacscript:14]:
```suggestion
repology=("project: yad")
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/pdf-mix-tool-git/pdf-mix-tool-git.pacscript (hunk 1 above)
Comment (echometerain) [reply packages/pdf-mix-tool-git/pdf-mix-tool-git.pacscript:10]: wdym?

Comment (echometerain) [reply packages/pdf-mix-tool-git/pdf-mix-tool-git.pacscript:10]: oh

Comment (echometerain) [reply packages/pdf-mix-tool-git/pdf-mix-tool-git.pacscript:10]: git package

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**